docs/docs/manual/grelfunctions.md

```diff
@@ -0,0 +1,537 @@
+
+#### Substring
+
+###### substring(s, n from, n to (optional))
+
+Returns the substring of s starting from character index from, and up to (excluding) character index to. If the to argument is omitted, substring will output to the end of s. For example, `"profound".substring(3)` returns the string “found”, and `"profound".substring(2, 4)` returns the string “of”.
+
+Character indexes start from zero. Negative character indexes count from the end of the string. For example, `"profound".substring(1, -1)` returns the string rofoun.
+
+###### slice(s, n from, n to (optional))
+
+Identical to substring() in relation to strings. Also works with arrays; see [Array functions section](#slicea-n-from-n-to-optional).
+
+###### get(s, n from, n to (optional))
+
+Identical to substring() in relation to strings. Also works with named fields. Also works with arrays; see [Array functions section](#geta-n-or-s-from-n-to-optional).
+
+#### Find and replace
+
+###### indexOf(s, sub)
+
+Returns the first character index of sub as it first occurs in s; or, returns -1 if s does not contain sub. For example, `"internationalization".indexOf("nation")` returns 5, whereas `"internationalization".indexOf("world")` returns -1.
+
+###### lastIndexOf(s, sub)
+
+Returns the first character index of sub as it last occurs in s; or, returns -1 if s does not contain sub. For example, `"parallel".lastIndexOf("a")` returns 3 (pointing at the second "a").
+
+###### replace(s, s or p find, s replace)
+
+Returns the string obtained by replacing the find string with the replace string in the inputted string. For example, `"The cow jumps over the moon and moos".replace("oo", "ee")` returns the string “The cow jumps over the meen and mees”. Find can be a regex pattern; if so, replace can also contain capture groups declared in find. 
+
+You cannot find or replace nulls with this, as null is not a string. You can instead:
+
+1. Facet by null and then bulk-edit them to a string, or
+2. Transform the column with an expression such as `if(value==null,'new',value)`
+
+###### replaceChars(s, s find, s replace)
+
+Returns the string obtained by replacing a character in s, identified by find, with the corresponding character identified in replace. For example, `"all the big cows lumber".replaceChars("aeiou", "A!IOU")` returns the string “All th! bIg cOws lUmb!r”. You cannot use this to replace a single character with more than one character.
+
+###### match(s, p)
+
+Attempts to match the string s in its entirety against the [regex](#grel-supported-regex) pattern p and, if the pattern is found, returns an array of the desired substrings (found in order). For example, `"230.22398, 12.3480".match(/.*(\d\d\d\d)/)` returns an array of 1 substring: [3480]. It does not find 2239 as the first sequence with four digits, because the regex indicates the four digits must come at the end of the string.
```

Comment:
> match(s, p)#

Attempts to match the string s in its entirety against the regex pattern p and, if the pattern is found, outputs an array of all capturing groups (found in order). For example, "230.22398, 12.3480".match(/.*(\d\d\d\d)/) returns an array of 1 substring: [ "3480" ]. It does not find 2239 as the first sequence with four digits, because the regex indicates the four digits must come at the end of the string.
> You will need to convert the array to a string to store it in a cell, with a function such as toString(). An empty array [] is returned when there is no match to the desired substrings. A null is output when the entire regex does not match.
> Remember to enclose your regex in forward slashes, and to escape characters and use parentheses as needed. Parentheses are required to denote a desired substring (capturing group); for example, “.*(\d\d\d\d)” would return an array containing a single value, while “(.*)(\d\d\d\d)” would return two. So, if you are looking for a desired substring anywhere within a string, use the syntax value.match(/.*(desired-substring-regex).*/).

I really don't love the phrase "capture group" or "capturing group" - it's really hard to work it in as though it's casual language. Maybe I can just link to a definition somewhere?